src/main/java/taxi/controller/driver/AddDriverController.java

```diff
@@ -21,11 +21,28 @@ public void doGet(HttpServletRequest req, HttpServletResponse resp)
     }
 
     @Override
-    public void doPost(HttpServletRequest req, HttpServletResponse resp) throws IOException {
+    public void doPost(HttpServletRequest req, HttpServletResponse resp)
+            throws IOException, ServletException {
         String name = req.getParameter("name");
         String licenseNumber = req.getParameter("license_number");
-        Driver driver = new Driver(name, licenseNumber);
-        driverService.create(driver);
-        resp.sendRedirect(req.getContextPath() + "/drivers/add");
+        String login = req.getParameter("login");
+        String password = req.getParameter("password");
+        String repeatPassword = req.getParameter("repeat_password");
+        if (password.equals(repeatPassword)) {
+            Driver driver = new Driver(name, licenseNumber, login, password);
+            driverService.create(driver);
+            sendMessage("sucMsg", "Driver: " + name + ", created successfully!"
+                    + " Driver with login: " + login + " can login.", req, resp);
+        } else {
+            sendMessage("errMsg", "Passwords don't match, please try again.", req, resp);
+        }
```

Comment:
Let's refactor this part and move this logic to the authentication service, and have a some method called `User register(username, password, repeatPassword) throws AuthenticationException;`